lib/os.js

```diff
@@ -66,17 +65,25 @@ function getCheckedFunction(fn) {
   });
 }
 
+const {
+  type,
+  release,
+  version
+} = getInformation();
+
 const getHomeDirectory = getCheckedFunction(_getHomeDirectory);
 const getHostname = getCheckedFunction(_getHostname);
 const getInterfaceAddresses = getCheckedFunction(_getInterfaceAddresses);
-const getOSRelease = getCheckedFunction(_getOSRelease);
-const getOSType = getCheckedFunction(_getOSType);
+const getOSRelease = () => release;
+const getOSType = () => type;
+const getOSVersion = () => version;
 
 getFreeMem[SymbolToPrimitive] = () => getFreeMem();
```

Comment:
(not directly related to this pr) what is the rationale of having these symbols? there is no comment and they aren't mentioned in our docs.

Reply:
@devsnek These were added in 473572ea2512cbe71e3423f8094657986352c32d… I don’t think there’s any deep meaning here, though

Reply:
These were added as a very simple convenience...

```js
> os.release
[Function: hidden] {
  [Symbol(Symbol.toPrimitive)]: [Function (anonymous)]
}
> os.release()
'10.0.19564'
> '' + os.release    // uses the toPrimitive...
'10.0.19564'
>
```